scripts/create_distr.sh

```diff
@@ -15,9 +15,10 @@
 # limitations under the License.
 #
 
+PACKAGE_NAME=pygridgain
 DISTR_DIR="$(pwd)/distr/"
 SRC_DIR="$(pwd)"
-DEFAULT_DOCKER_IMAGE="quay.io/pypa/manylinux2010_x86_64"
+DEFAULT_DOCKER_IMAGE="quay.io/pypa/manylinux2014_x86_64"
```

Comment:
* `manylinux2014_x86_64` and `manylinux2014_i686` strings are repeated
* `quay.io/pypa/manylinux2014_x86_64` is a constant here, but `quay.io/pypa/manylinux2014_i686` is hardcoded below

I feel like we could clean this up a bit with more constants.

Reply:
Agree.

Reply:
Fixed.